packages/uikit-react-native-foundation/src/ui/GroupChannelMessage/Message.file.image.tsx

```diff
@@ -10,20 +10,32 @@ import createStyleSheet from '../../styles/createStyleSheet';
 import useUIKitTheme from '../../theme/useUIKitTheme';
 import MessageContainer from './MessageContainer';
 import type { GroupChannelMessageProps } from './index';
+import { CustomComponentContext } from '../../context/CustomComponentCtx';
 
 const ImageFileMessage = (props: GroupChannelMessageProps<SendbirdFileMessage>) => {
   const { onPress, onLongPress, variant = 'incoming' } = props;
+  const ctx = useContext(CustomComponentContext);
 
   const { colors } = useUIKitTheme();
 
+  const content = (
+    <>
+      <ImageWithPlaceholder source={{ uri: getThumbnailUriFromFileMessage(props.message) }} style={styles.image} />
+      {props.children}
+    </>
+  );
+
   return (
     <MessageContainer {...props}>
-      <Box style={styles.container} backgroundColor={colors.ui.groupChannelMessage[variant].enabled.background}>
-        <PressBox activeOpacity={0.8} onPress={onPress} onLongPress={onLongPress}>
-          <ImageWithPlaceholder source={{ uri: getThumbnailUriFromFileMessage(props.message) }} style={styles.image} />
-        </PressBox>
-        {props.children}
-      </Box>
+      <PressBox activeOpacity={0.8} onPress={onPress} onLongPress={onLongPress}>
+        {
+          ctx?.renderGenericMessage ? ctx.renderGenericMessage({ content }) : (
+            <Box style={styles.container} backgroundColor={colors.ui.groupChannelMessage[variant].enabled.background}>
+              {content}
+            </Box>
+          )
+        }
+      </PressBox>
```

Comment:
A bit of a behavior change here. The `Box` and `PressBox` components are swapped and `props.children` are now inside the `PressBox`. I guess we won't be using the default functionality any more, so it may not matter, but does this change the default behavior a lot?

Reply:
yea the `props.children`, effectively become part of the touchable opacity area. I'll move it out